iphone/TitaniumKit/TitaniumKit/Sources/API/WebFont.m

```diff
@@ -34,25 +34,10 @@ - (BOOL)isSizeNotSet
   return size < 4;
 }
 
-- (BOOL)isValidTextStyle:(NSString *)theStyle
-{
-  return (
-      [theStyle isEqualToString:UIFontTextStyleBody] ||
-      [theStyle isEqualToString:UIFontTextStyleCaption1] ||
-      [theStyle isEqualToString:UIFontTextStyleCaption2] ||
-      [theStyle isEqualToString:UIFontTextStyleHeadline] ||
-      [theStyle isEqualToString:UIFontTextStyleSubheadline] ||
-      [theStyle isEqualToString:UIFontTextStyleFootnote] ||
-      [theStyle isEqualToString:UIFontTextStyleCallout] ||
-      [theStyle isEqualToString:UIFontTextStyleTitle3] ||
-      [theStyle isEqualToString:UIFontTextStyleTitle2] ||
-      [theStyle isEqualToString:UIFontTextStyleTitle1]);
-}
-
 - (UIFont *)font
 {
   if (font == nil) {
-    if (textStyle != nil && [self isValidTextStyle:textStyle]) {
+    if (textStyle != nil && [textStyle isKindOfClass:[NSString class]]) {
```

Comment:
Any particular reason you removed this check? What will happen if the text style is not one of the valid constants?